connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConfig.java

```diff
@@ -109,6 +109,11 @@
                     " from the domain of the REST API.";
     protected static final String ACCESS_CONTROL_ALLOW_ORIGIN_DEFAULT = "";
 
+    public static final String ACCESS_CONTROL_ALLOW_METHODS_CONFIG = "access.control.allow.methods";
+    protected static final String ACCESS_CONTROL_ALLOW_METHODS_DOC =
+        "Set value to Jetty Access-Control-Allow-Origin header for the specified methods. "
```

Comment:
Probably not relevant that we use Jetty under the hood. Also, this doesn't actually affect the `Access-Control-Allow-Origin` header, so the documentation is a bit confusing. Might just want to adjust to say it sets the methods supported for cross origin requests by setting the `Access-Control-Allow-Methods` header. The explanation of defaults is good.